Cask

```diff
@@ -0,0 +1,10 @@
+(source gnu)
+(source melpa)
+
+(package "flycheck-coala" "0.1" "Flycheck plugin for coala.io code analyzer")
+
+(development
+ (depends-on "dash")
+ (depends-on "flycheck")
+ (depends-on "f")
+ (depends-on "ert-runner"))
```

Comment:
Hi,

it looks like you're missing a railing newline before the end of the file here. [This article](http://stackoverflow.com/a/5813359/3212182) has more information about this issue.